language_dump.py

```diff
@@ -40,16 +44,36 @@
             if language_to_extract in data['strings'][string_key]:
                 if verbose:
                     print("Found existing translation for " + data['id'])
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][language_to_extract]
+                strings_by_object[data['id']][string_key] = \
+                    data['strings'][string_key][language_to_extract]
```

Comment:
This feels a little awkward due to the wrapping. Perhaps save the rvalue first with a local (alias) assignment.